extensions/passwords.py

```diff
@@ -0,0 +1,38 @@
+# Nextcord
+from nextcord.ext import commands
+
+# Randoms
+import random
+
+# Variabless
+lower = 'abcdefghijklmnopqrstuvwxyz'
+upper = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+numbers = '0123456789'
+symbols = '!@#$%^&'
+
+
+# commands
+class Passwords(commands.Cog):
+    def __init__(self, bot):
+        self.bot: commands.Bot = bot
+
+    @commands.command(
+        aliases=['pass', 'word']
+    )
+    async def password(self, ctx: commands.Context):
+        all = lower + upper + numbers + symbols
+        length = 9
+        password = "".join(random.sample(all, length))
+        await ctx.send(
+            f'This is your password: `{password}`,'
+            " if you don't like it,"
+            ' or if you want to make changes,'
+            ' you can visit:'
+            ' https://react-password-generator-'
+            'seven.vercel.app/'
+            '. This page was made by: Titoyan, who is my creator'
+        )
```

Comment:
Try to keep this simple, don't say it all in one message

Reply:
How would you like it to look

Reply:
I did it that way, because you told me you were going to change it.

Reply:
Yes but, let's keep it simple until I change it.

Reply:
ok, I will change it